### PR TITLE
Table visualization with column named "children" renders +/- buttons

### PR DESCRIPTION
**Pull request by kravets-levko**
## What type of PR is this? (check all applicable)

- [x] Bug Fix

## Related Tickets & Documents

Fixes getredash/redash#4393

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

Before:
image

After:
image

### CODE REVIEW

**Comment by gabrieldutra** [comment]
Yeah, it is supposed to be a very simple fix, I missed the part of the docs that mentioned `childrenColumnName` when I first read it :sweat_smile:. But that surprisingly still holds the issue:

image

In the docs it seem that prop is an string array, so perhaps `childrenColumnName={[]}`

File: client/app/visualizations/table/Renderer.jsx [hunk 1]
```diff
@@ -65,6 +65,7 @@ export default function Renderer({ options, data, context }) {
         data-test="TableVisualization"
         columns={tableColumns}
         dataSource={preparedRows}
+        childrenColumnName={null}
```
Comment (gabrieldutra) [inline client/app/visualizations/table/Renderer.jsx]:
```suggestion
        childrenColumnName={[]}
```

Comment (kravets-levko) [reply client/app/visualizations/table/Renderer.jsx]: @gabrieldutra 

image

Comment (gabrieldutra) [reply client/app/visualizations/table/Renderer.jsx]: :man_facepalming:

Comment (gabrieldutra) [reply client/app/visualizations/table/Renderer.jsx]: That worked after I update Antd, maybe they changed it to allow array recently?

Comment (kravets-levko) [reply client/app/visualizations/table/Renderer.jsx]: `childrenName` prop is [declared as optional/nullable](https://github.com/ant-design/ant-design/blob/master/components/table/interface.tsx#L212), but later there are no any conditions - it's just used as is - e.g. `item[this.props.childrenName]` - implicit cast to string. I'll try to find a new solution.

**Pull request merged**